docs/GETTING_STARTED.md

```diff
@@ -0,0 +1,271 @@
+```
+We can inspect one of these ztoc's with the following command (you will need to
+replace the digest with one of the ones created above) : 
+```
+sudo ./soci ztoc info sha256:4c1d63f476d4907e0db42b8736f578e79432a28d304935708c918c95e0e4df00
+```
+
+This will print to STDOUT the ztoc, which contains all of the information that
+soci needs to find a given file in the compressed tar.gz.  We can also view the
+index manifests by running : 
+
+```
+sudo ./soci index list
+```
+
+This will list out all of our index manifests.  To inspect an individual one we
+can use following (as always replace the digest with your own) : 
+
+```
+sudo ./soci index info sha256:f5f2a8558d0036c0a316638c5575607c01d1fa1588dbe56c6a5a7253e30ce107
+```
+
+This will dump out the index manifest in json.
+
+### Pushing the manifest to the registry
+Next we need to push the manifest to the registry with the following command.
+Just like with ctr we need to use `--plain-http` flag : 
+
+```
+sudo ./soci push --plain-http localhost:5000/rabbitmq:latest
+```
+
+This will push all of the ztoc files and the soci index manifest.
+
+## Running the image
+
+### Configuring containerd
+To have containerd use the soci-snapshotter we need to reconfigure it.  
+Containerd is a daemon managed using the systemd / systemctl tools.  
+First let's stop containerd
+
+```
+sudo systemctl stop containerd
+```
+
+Next we need to edit the config file that manages all of containerd's settings.  
+This file is /etc/containerd/config.toml.  By default the file has an example
+configuration that is all commented out (please see all of the # signs). Let's 
+edit this file to include soci snapshotter as a plug in with the following:
+
+```
+[proxy_plugins]
+  [proxy_plugins.soci]
+    type = "snapshot"
+    address = "/run/soci-snapshotter-grpc/soci-snapshotter-grpc.sock"
+```
+
+These lines tell containerd that there is a snapshot plugin named soci and to 
+communicate with it via a socket file.  Now let's restart containerd
+
+```
+sudo systemctl restart containerd
+```
+
+To confirm that containerd knows about soci run
+
+```
+ctr plugin ls
+```
+
+This will list all of the plugins confirm that there is a **soci** 
+plugin of type **io.containerd.snapshotter.v1**
+
+### Preparing soci snapshotter
+First we need to start the snapshotter binary, it lives in the `out` directory.
+In the example command we arbitrarily are pushing the logs to a file named
+`~/soci-snapshotter-logs`, but please feel free to choose a different location : 
+
+```
+sudo ./soci-snapshotter-grpc&> ~/soci-snapshotter-logs &
+```
+
+Once the snapshotter is running we can call the rpull command from the soci cli.
+This command reads the manfiest from the registry and mounts FUSE filesystems
+for each layer.   
+
+In this command the `--soci-index-digest` needs to be the digest of the soci
+index manifest.  The final argument is the tag of the image : 
```

Comment:
Maybe change `tag` to `image reference`? I'm being pedantic, but `tag` would refer to just 'latest'.